packages/core/docs/commercetools/configuration.md

```diff
@@ -18,7 +18,38 @@ Commercetools configuration is located in two places:
 }]
 ```
 
-- `useNuxtI18nConfig` - when this property is set to true, `@vue-storefront/commercetools/nuxt` package will use `i18n` config object provided in `nuxt.config.js`. When set to false, `i18n` config should be declared directly inside this package configuration. You can read more about `i18n` config in Vue Storefront [here](../advanced/internationalization.md)
+- `useNuxtI18nConfig` - when this property is set to true, `@vue-storefront/commercetools/nuxt` package will use `i18n` config object provided in `nuxt.config.js`. When set to false, `i18n` config should be declared directly inside this package configuration. You can read more generally about `i18n` config in Vue Storefront [here](../advanced/internationalization.md). On the other hand, there are things which are specific for the commercetools, you can read about them below.
+
+### Adding states for the country
```

Comment:
What I'm missing is

- the use case: "Adding states for the country" is not a use case because I don't know the context (checkout/profile etc)
-  there is no info where I can access this data in my app, there is only a mention about a single usage of this data on the UI ut no mention on how to retrieve this data actually (will it be available in some composales or through $i18n from useContext etc)